client/state/data-layer/wpcom/sites/automated-transfer/status/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	EVERY_FIVE_SECONDS,
+	add as addRunner,
+	remove as removeRunner,
+} from 'lib/interval/runner';
+import wpcom from 'lib/wp';
+
+import {
+	AUTOMATED_TRANSFER_STATUS_REQUEST as REQUEST,
+	AUTOMATED_TRANSFER_STATUS_UPDATE as UPDATE,
+} from 'state/action-types';
+
+import { requestStatus, updateStatus } from 'state/automated-transfer/actions';
+import { transferStates } from 'state/automated-transfer/constants';
+
+const watchList = new Map();
+
+const statusMapping = {
+	complete: transferStates.COMPLETE,
+	uploading: transferStates.START,
+};
```

Comment:
This is _the_ spot in Calypso to jump to in order to understand how the data in the API maps onto Calypso idioms. Although you created those constants to match the API when you wrote them, we really don't have any kind of guarantee that they'll stay that way. This brings a level of isolation and quick debugging when things eventually break. Fix it here and fix it everywhere.